contracts/exchange/test/dispatcher.ts

```diff
@@ -238,37 +238,35 @@ describe('AssetProxyDispatcher', () => {
             expect(newBalances).to.deep.equal(erc20Balances);
         });
 
-        it('should perform a noop transfer if from == to', async () => {
```

Comment:
Why are we removing this test?

Reply:
We actually no longer noop in this case. It wasn't a very good test if it was still passing.